docs/source/starter/introduction_guide.rst

```diff
@@ -127,10 +133,40 @@ Now we add the training_step which has all our training loop logic
             loss = F.nll_loss(logits, y)
             return loss
 
+Optimizer
```

Comment:
why is this moved? :rabbit:

Reply:
Hi @Borda. Thanks for the review!

If the original order that "Optimizer" section is after the "Data" section, the code in the "Data" section fails to execute. This is because the required configure_optimizers method is defined in the "Optimizer" section after the "Data" section.

So it may be good that we swap the order with "Data" section and "Optimizer" section to avoid the problem.

If it is not reasonable, I'll be happy to fix the change!